apps/myaccount/src/components/multi-factor-authentication/authenticators/totp-authenticator.tsx

```diff
@@ -261,7 +261,20 @@ export const TOTPAuthenticator: React.FunctionComponent<TOTPProps> = (
     const renderQRCode = (step: number): JSX.Element => {
         return (
             <Segment basic >
-                    <h5 className=" text-center" > { t(translateKey + "modals.scan.heading") }</h5>
+                { !isTOTPConfigured
+                    ?
+                    <Grid>
+                        <Grid.Row columns={ 1 } centered={ true }>
+                            <Grid.Column mobile={ 12 } tablet={ 12 } computer={ 12 } >
+                                <Message info>
+                                        { "QR code has been successfully added to your profile!" }
```

Comment:
Shall we localize this?

Reply:
Fixed with b5e82899c9c6b76d2ba7e0e0e5e41d10436e5bc0